debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -441,7 +459,7 @@ protected Object convertArray(Column column, Field fieldDefn, Object data) {
         if (!(data instanceof List)) {
             return handleUnknownData(column, fieldDefn, data);
         }
-        return data;
+        return ((List<?>)data).stream().map(elementConverter::convert).collect(Collectors.toList());
```

Comment:
I'd rather wrap that, so that one method invocation is per line. It's recommended for stream operations, as it's otherwise impossible to step though in the debugger:

```java
((List<?>)data).stream()
    .map(elementConverter::convert)
    .collect(Collectors.toList());
```